pandas/tests/extension/base/ops.py

```diff
@@ -263,3 +263,10 @@ def test_unary_ufunc_dunder_equivalence(self, data, ufunc):
         else:
             alt = ufunc(data)
             tm.assert_extension_array_equal(result, alt)
+
+
+class BaseRoundingTests:
```

Comment:
can this go into one of the existing test classes?

Reply:
moved to methods